### PR TITLE
Add documentation for the project properties' sensors panel

### PR DESCRIPTION
**Pull request by nirvn**
This PR adds documentation covering the new project properties' sensors panel (an upcoming feature in QGIS 3.32).

### CODE REVIEW

**Comment by nirvn** [comment]
@DelazJ , this is ready for review and merging.

**Review by DelazJ — COMMENTED** [review]
Thanks @nirvn (sorry I forgot to publish my review)
Just some (GUI label) formatting suggestions.

**Comment by nirvn** [comment]
@DelazJ , thanks for the review, comments have been addressed.

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 1]
```diff
@@ -1947,7 +1947,7 @@ python scripts;
 
    Sensor Settings sub-panel
 
-Once a sensor is configured, you can use the |start| :guilabel:`Start` button to connect the
+Once a sensor is configured, you can use the |start| :sup:`Start` button to connect the
```
Comment (nirvn) [inline docs/user_manual/introduction/qgis_configuration.rst:1950]: @DelazJ , TIL :)

**Comment by DelazJ** [comment]
https://github.com/qgis/QGIS-Documentation/actions/runs/4902208515/jobs/8753918834?pr=8194#step:7:338
> home/runner/work/QGIS-Documentation/QGIS-Documentation/docs/user_manual/introduction/qgis_configuration.rst:1939: WARNING: Bullet list ends without a blank line; unexpected unindent.

I have no idea what this is about; I can see a blank line at line 1942. So should be good. Anyone sees something?

File: docs/user_manual/introduction/qgis_configuration.rst [hunk 2]
```diff
@@ -1914,6 +1914,44 @@ These settings can be overwritten from the 3D map :ref:`configuration
...
+.. figure:: img/project_sensors.png
+   :align: center
+
+   Project Sensors tab
+
+To add a new sensor, click on the |symbologyAdd| button. A settings sub-panel
+will open and allow you to configure:
+
+* the :guilabel:`Sensor name`: used to retrieve sensor values in expressions and
+python scripts;
```
Comment (DelazJ) [inline docs/user_manual/introduction/qgis_configuration.rst:1939]:
```suggestion
  python scripts;
```
Got the culprit! much easier to find on computer than on a phone 😄

**Pull request merged**